test/e2e/BrsMock.test.js

```diff
@@ -46,4 +46,12 @@ describe("end to end brightscript functions", () => {
             "name",
         ]);
     });
+
+    test("mockFunctions.brs", async () => {
+        await execute([resourceFile("mockFunctions.brs")], outputStreams);
+
+        expect(allArgs(outputStreams.stdout.write).filter(arg => arg !== "\n")).toEqual([
```

Comment:
🤔 I think there's been maybe… one time where we didn't do `.filter(arg => arg !== "\n")`.  @underwoo16 think it's worth us looking into abstracting this a bit more in the future? The boilerplate isn't too bad and it's definitely clear what's happening, but it's still boilerplate.